### PR TITLE
Add guides page for inspector component tab

### PR DESCRIPTION
**Pull request by Bestra**
Adds a new page for the inspector component tree.  I tried to keep it pretty similar to the existing view tree documentation.

Here are screeshots of the content:

screen shot 2018-04-27 at 1 25 02 pm

screen shot 2018-04-27 at 1 25 10 pm

image

### CODE REVIEW

File: guides/v3.1.0/ember-inspector/component-tree.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+The Components tab displays a collapsible representation of the views and components that are currently being rendered.  Selecting a component from the tree will open it in the [Object Inspector](../object-inspector/). 
+
+<img src="/images/guides/ember-inspector/component-tree-intro.png" width="680">
+
+Components will be displayed with curly braces.  Views are grayed out and not selectable.  Use the [View Tree](../view-tree) to get more information on Views that are being rendered.
```
Comment (mansona) [inline guides/v3.1.0/ember-inspector/component-tree.md]: This link is missing the trailing slash, it should be

```
[View Tree](../view-tree/) 
``` 

and then it should fix the Travis build 👍

**Comment by Bestra** [comment]
@mansona 🤦‍♂️  I'll remember to actually run the tests before pushing next time 😄

**Comment by mansona** [comment]
@Bestra I probably shouldn't document this is such a public forum but I often rely on Travis for these initial checks 😂

**Comment by jenweber** [comment]
I love this feature/guides addition.

@mansona want to go ahead and merge?

**Pull request merged**